### DISCUSSION TITLE
BUG: "createCommitOnBranch" link goes to wrong section

### DISCUSSION THREAD

**Discussion by danawoodman** [community/community#28430] [Apps, API and Webhooks]
On the [GraphQL mutations docs page](https://docs.github.com/en/graphql/reference/mutations) the section `createCommitOnBranch` has an anchor tag to `#addStar` which causes you to jump up to the `addStar` section and not `createCommitOnBranch`. Looks like the anchor link and ID need to be updated.

![Screen Shot 2022-08-04 at 6 51 18 PM](https://user-images.githubusercontent.com/157695/182984738-32ce6aa7-c4ba-47d4-a575-5941a058b3b0.png)

**Comment by radar-blip** [comment]
For me, its pointing to deleteTeamDiscussion. 

Anyway, you can file a Bug Report if you are so inclinded, here:
https://support.github.com/contact/bug-report

**Reply by danawoodman** [reply]
Weird, maybe it is auto-generating them on request or something weird. I'll submit a bug report

**Reply by radar-blip** [reply]
Yes, something weird happening there. Anyway, hopefully they will get it fixed.
Have a nice flight!
